reinforcement_learning/rl_cartpole_ray/rl_cartpole_ray.ipynb

```diff
@@ -0,0 +1,564 @@
+{
```

Comment:
<p>Was this cell tested?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1331/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Yes, the graph is generated for rewards for each episode.</p>